models/Comment.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+
+const commentSchema = new Schema({
+    : {
+        type: String,
+        required: true
```

Comment:
Can you add field name here, 
e.g: 
comment: {
   type: String,
   required: true
}

Reply:
I know of this. I just want to leave it blank for now. Until I add comment section to front-end later. If that's okay,

Reply:
I think you can have these fields here then, on the frontend you will put the titles you have in the comment model